common/rpc/interceptor/namespace_logger.go

```diff
@@ -74,7 +74,7 @@ func (nli *NamespaceLogInterceptor) Intercept(
 				certThumbprint = fmt.Sprintf("%x", md5.Sum(cert.Raw))
 			}
 		}
-		nli.logger.Info(
+		nli.logger.Debug(
 			"frontend method invoked",
```

Comment:
nit (and can be ignored):
```suggestion
			"Frontend method invoked.",
```